### PR TITLE
Updated the quickstart guide for Navarino release

### PR DESCRIPTION
**Pull request by amanr032**
What type of PR is this?

> /kind documentation

What this PR does / why we need it:
 Documentation for the change of installation steps 

Which issue(s) this PR fixes:
Fixes 

Test Report Added?:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

Test Report:

### CODE REVIEW

File: content/soda-gettingstarted/quickstart.md [hunk 1]
```diff
@@ -25,72 +26,56 @@ To have more control of the installation configurations, or try out various conf
...
-## Quick Installation and experience of SODA releases
 
-{{% notice info %}}
-**Ubuntu 16.04 support for SODA is deprecated as of SODA Jerba release v1.4.0.** <br />
-**If you have a requirement to install SODA on Ubuntu 16.04, please contact us on slack and we will try to help with the setup.**
-{{% /notice %}}
+## Quick Installation and experience of SODA releases
 
-This procedure applies to 18.04 Linux distributions.
+This procedure applies to 18.04 & 20.04 Linux distributions.
```
Comment (anvithks) [inline content/soda-gettingstarted/quickstart.md:33]: Ubuntu 18.04 & 20.04

File: content/soda-gettingstarted/quickstart.md [hunk 2]
```diff
@@ -25,72 +26,56 @@ To have more control of the installation configurations, or try out various conf
...
-		 - export HOST_IP = <your_real_host_ip>
-		 - vim group_vars/common.yml # Modify `host_ip` address if needed
+```bash
+chmod +x install_dependencies.sh && source install_dependencies.sh
+export PATH=$PATH:/home/$USER/.local/bin
+#The HOST_IP need to be set to the real_host_ip of your host (ex: 192.168.1.10 or 127.0.0.1 for localhost)
+export HOST_IP = <your_real_host_ip>
+vim group_vars/common.yml # Modify `host_ip` address if needed
+```
+     Enable following configurations
```
Comment (anvithks) [inline content/soda-gettingstarted/quickstart.md:67]: This only enables delfin. What about Multicloud and Terra?

File: content/soda-gettingstarted/quickstart.md [hunk 3]
```diff
@@ -25,72 +26,56 @@ To have more control of the installation configurations, or try out various conf
...
+      `sudo apt-get update && sudo apt-get install -y git`
 
 	- **Get and Install Release Binaries**
 	
-	{{% notice info %}}
 
-	Please replace RELEASE_TAG with the corresponding value from Release Tag table above (ex: v1.6.0 etc)
 
-	{{% /notice %}}
+**_NOTE:_**  The note content.Please replace RELEASE_TAG with the corresponding value from Release Tag table above (ex: v1.8.0 etc)
```
Comment (anvithks) [inline content/soda-gettingstarted/quickstart.md:45]: Use the notice shortcode.

**Review by anvithks — CHANGES_REQUESTED** [review]
Please check the review comments.

File: content/soda-gettingstarted/quickstart.md [hunk 4]
```diff
@@ -25,72 +26,56 @@ To have more control of the installation configurations, or try out various conf
...
+
+
+```bash
+wget  [https://github.com/sodafoundation/installer/releases/download/<RELEASE_TAG>/installer-[RELEASE_TAG].tar.gz](https://github.com/sodafoundation/installer/releases/download/[RELEASE_TAG]/installer-[RELEASE_TAG].tar.gz)
```
Comment (anvithks) [inline content/soda-gettingstarted/quickstart.md:49]: Change the Markdown URL format to simple URL [https://github.com/sodafoundation/installer/releases/download/<RELEASE_TAG>/installer-[RELEASE_TAG].tar.gz](https://github.com/sodafoundation/installer/releases/download/[RELEASE_TAG]/installer-[RELEASE_TAG].tar.gz) - > https://github.com/sodafoundation/installer/releases/download/[RELEASE_TAG]/installer-[RELEASE_TAG].tar.gz

**Review by anvithks — DISMISSED** [review]
LGTM

File: content/soda-gettingstarted/quickstart.md [hunk 5]
```diff
@@ -25,72 +26,68 @@ To have more control of the installation configurations, or try out various conf
...
-	    -   Run following command `go version` and check the version
-	    -   If the required version is not installed on the system, install golang,1.13.9 using:
-		   - wget **[https://storage.googleapis.com/golang/go1.13.9.linux-amd64.tar.gz](https://storage.googleapis.com/golang/go1.13.9.linux-amd64.tar.gz)**
-		- tar -C /usr/local -xzf go1.13.9.linux-amd64.tar.gz
-		- echo export PATH=$PATH:/usr/local/go/bin >> /etc/profile
-		- echo export GOPATH=$HOME/gopath >> /etc/profile
-		- source /etc/profile
-
-	- **Get and Install Release Binaries**
+	- **Terra**
```
Comment (joseph-v) [inline content/soda-gettingstarted/quickstart.md:78]:
```suggestion
	- **Terra (SODA Hotpot)**  
```

Comment (amanr032) [reply content/soda-gettingstarted/quickstart.md]: updated

**Review by joseph-v — DISMISSED** [review]
LGTM

File: content/soda-gettingstarted/quickstart.md [hunk 6]
```diff
@@ -25,72 +26,68 @@ To have more control of the installation configurations, or try out various conf
...
+         
+```bash
+chmod +x install_dependencies.sh && source install_dependencies.sh
+export PATH=$PATH:/home/$USER/.local/bin
+#The HOST_IP need to be set to the real_host_ip of your host (ex: 192.168.1.10 or 127.0.0.1 for localhost)
+export HOST_IP = <your_real_host_ip>
+vim group_vars/common.yml # Modify `host_ip` address if needed
+```
+
+- **Enable configurations to install Soda Products**
```
Comment (anvithks) [inline content/soda-gettingstarted/quickstart.md:68]: nit: Soda -> SODA

File: content/soda-gettingstarted/quickstart.md [hunk 7]
```diff
@@ -25,72 +26,68 @@ To have more control of the installation configurations, or try out various conf
...
+	- **DELFIN**  
+		- In file installer/ansible/group_vars/delfin.yml  `enable_delfin: true`
+		- In file installer/ansible/group_vars/srm-toolchain.yml `install_srm_toolchain: true`
+		- In file installer/ansible/group_vars/dashboard.yml `enable_dashboard: true`
+
+	- **STRATO (SODA MultiCloud)**
```
Comment (anvithks) [inline content/soda-gettingstarted/quickstart.md:74]: nit: MultiCloud -> Multicloud

**Review by anvithks — DISMISSED** [review]
Some minor nits. Please fix the review comments and then merge the PR.
LGTM

File: content/soda-gettingstarted/quickstart.md (hunk 6 above)
Comment (amanr032) [reply content/soda-gettingstarted/quickstart.md]: updated

File: content/soda-gettingstarted/quickstart.md (hunk 7 above)
Comment (amanr032) [reply content/soda-gettingstarted/quickstart.md]: updated

**Review by anvithks — APPROVED** [review]
LGTM

**Review by joseph-v — APPROVED** [review]
LGTM

**Pull request merged**